include/kos/thread.h

```diff
@@ -189,7 +231,7 @@ typedef __attribute__((aligned(32))) struct kthread {
     uint32_t *stack;
 
     /** \brief  Size of the thread's stack, in bytes. */
-    uint32_t stack_size;
+    size_t stack_size;
```

Comment:
Did you mean `usize_t` here? Or is there a reason to have it signed?

Reply:
`size_t` is supposed to be unsigned? I thought `ssize_t` was the (nonstandard) signed equivalent?

Reply:
`size_t` is unsigned, I don't think `usize_t` is a standard thing?

Reply:
Woops my bad! Think I got mixed it up with Rust types  :sweat_smile: